blocks/amp-story/amp-story-cta-layer.js

```diff
@@ -105,9 +110,25 @@ export default registerBlockType(
 			}
 
 			componentDidMount() {
+				const rootClientID = getBlockRootClientId( this.props.clientId );
+				const blockIndex = wp.data.select( 'core/editor' ).getBlockIndex( rootClientID );
+				let noticeMessage = null;
+				const noticeOptions = {
+					id: 'amp-errors-notice-removed-cta'
+				};
+
 				if ( this.props.attributes.hasMultipleCtaBlocks ) {
 					removeBlock( this.props.clientId );
-					dispatch( 'core/editor' ).createWarningNotice( __( 'Multiple CTA Layers are not allowed, the block was removed.', 'amp' ) );
+					noticeMessage = wp.i18n.__( 'Multiple CTA Layers are not allowed, the block was removed.', 'amp' );
+				}
+
+				if ( 0 === blockIndex ) {
+					removeBlock( this.props.clientId );
+					noticeMessage = wp.i18n.__( 'CTA layer is not allowed on the first page, the block was removed.', 'amp' );
```

Comment:
Actually CTA Layer is allowed on each page -- it has to be the last Layer of a Page, so it can't be the first Layer in case there is another Layer already existing.

Reply:
For reference: https://www.ampproject.org/docs/reference/components/amp-story#amp-story-cta-layer

Reply:
Hmm yeah that is interesting, I wonder why I got that console error then. I'll test again and see what happens.

Reply:
I tried again and got the same error, it is the last layer on the first page:
![image](https://user-images.githubusercontent.com/3016425/48917282-1bcbff00-ee6d-11e8-88c5-cc206d0d2fa6.png)

Reply:
Okay, looks like they have updated AMP but didn't update the docs accordingly.

Thanks for finding and adding this!

Probably it would be good to hide the CTA Layer Icon via CSS from the Inserter of the first Page then, too.